activities/Clock.activity/js/activity.js

```diff
@@ -94,8 +94,27 @@ define(["sugar-web/activity/activity","sugar-web/env","sugar-web/graphics/radiob
                 that.updateSizes();
                 that.drawBackground();
             };
+            // Switch to full screen when the full screen button is pressed
+            document.getElementById("fullscreen-button").addEventListener('click', function() {
+              document.getElementById("main-toolbar").style.display = "none";
+              document.getElementById("canvas").style.top = "0px";
+              document.getElementById("unfullscreen-button").style.visibility = "visible";
```

Comment:
Please respect indentation of original file you use 2 spaces files uses 4. Same for below.